src/ert/gui/simulation/experiment_panel.py

```diff
@@ -297,3 +317,39 @@ def validationStatusChanged(self) -> None:
             self.run_button.text() == EXPERIMENT_READY_TO_RUN_BUTTON_MESSAGE
             and widget.isConfigurationValid()
         )
+
+    def populate_clipboard_debug_info(self) -> None:
+        kv = {"**Platform**": "", ":-----": ":-----"}
+        kv["Date"] = datetime.now().isoformat(" ", "seconds")
+        kv["OS"] = (
+            platform.system() + " " + platform.release() + " " + platform.machine()
+        )
+        kv["Hostname"] = platform.node()
+        kv["Komodo release"] = os.environ.get("KOMODO_RELEASE", "")
+        kv["Python version"] = platform.python_version()
+
+        kv["**Ensemble**"] = ""
+        kv["Queue"] = self.config.queue_config.queue_system.name.capitalize()
+        kv["Simulation mode"] = self.get_current_experiment_type().name()
+        kv["Config file"] = self._config_file
+        kv["Ensemble path"] = self.config.ens_path
+        kv["Ensemble size"] = str(self._model.ensemble_size)
+        kv["Max running"] = str(self.config.queue_config.queue_options.max_running)
```

Comment:
We could possibly list all the values in the queue_options class, or only non-defaulted (or if the latter is tricky, all the non-None values).

Reply:
I opted for the non-None variant. Looking at LSF-options there were mostly those variants. Other systems like Torque or Slurm had some extras, but we can rather expand on that if need be.